lib/fetch-server.js

```diff
@@ -108,12 +106,20 @@ export class FetchServer {
       }
     };
 
+    this.restore(); // clean up prior FetchServer stubs (if necessary)
```

Comment:
just a refactor, group all stub code together so the flow is easier to understand